discord_simde/members.tf

```
@@ -369,3 +369,15 @@ module nasser_chaouchi {
     (discord_role.integ.id) = true #Team info integ 21
   }
 }
+
+module thibaud_duhautbout {
+  source        = "./member"
+  api_token     = var.api_token
+  server_id     = discord_server.server_id
+  username      = "Thibaud"
+  discriminator = 5605
+  roles = {
+    (discord_role.member.id) = true
+    (discord_role.treso.id) = true
+  }
+}
```

La ptite ending new line ^^
```suggestion
}

```